### PR TITLE
Don't copy dictionary extensions we already pre-loaded from the system.

### PR DESCRIPTION
**Pull request by mmeeks**
Change-Id: I9cf7459c5214b6d7c23d5097822c7827409577a0

### CODE REVIEW

File: kit/Kit.cpp [hunk 1]
```diff
@@ -179,6 +179,8 @@ namespace
                 strcmp(path, "sdk") != 0 &&
                 strcmp(path, "debugsource") != 0 &&
                 strcmp(path, "share/basic") != 0 &&
+                strncmp(path,  "share/extensions/dict-", // preloaded
```
Comment (Ashod) [inline kit/Kit.cpp:182]: Is the hyphen at the end of 'dict' intentional?

**Review by Ashod — APPROVED** [review]
Looks good, one Q.

**Comment by mmeeks** [comment]
Sure - the dicts are all dict-en dict-DE etc. =)

**Comment by Ashod** [comment]
> Sure - the dicts are all dict-en dict-DE etc. =)

Yes, but what good is it if we don't compare it?
`sizeof("share/extensions/dict")` <= missing hyphen.

**Pull request merged**